src/pages/selectSalesOfferPackage.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { ReactElement } from 'react';
+import ErrorSummary from '../components/ErrorSummary';
+import FormElementWrapper from '../components/FormElementWrapper';
+import { FullColumnLayout } from '../layout/Layout';
+import { SALES_OFFER_PACKAGES_ATTRIBUTE } from '../constants';
+import { getSalesOfferPackagesByNocCode } from '../data/auroradb';
+import { SalesOfferPackage, CustomAppProps, ErrorInfo, NextPageContextWithSession } from '../interfaces';
+import { getNocFromIdToken } from '../utils';
+import CsrfForm from '../components/CsrfForm';
+import { redirectTo } from './api/apiUtils';
+import { getSessionAttribute } from '../utils/sessions';
+
+const pageTitle = 'Select Sales Offer Package - Fares Data Build Tool';
+const pageDescription = 'Sales Offer Package selection page of the Fares Data Build Tool';
+const errorId = 'sales-offer-package-error';
```

Comment:
Does this errorId actually link to any ID on the page?

Reply:
No